package.json

```diff
@@ -2,6 +2,10 @@
   "name": "helix-ui",
   "version": "0.0.1",
   "description": "Web Component Library",
+  "keywords": [
+    "custom elements v1",
```

Comment:
Are you sure you want `v1` in the string?

Also, possible alternative keywords: `design system`, `ui toolkit`

Reply:
I'll tweak.

Reply:
Didn't include `design system` because this is just one _part_ of a design system. I did add `ui toolkit` and broke `v1` into its own keyword.